test/ScriptExecutorTests/ScriptExecutorTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using System.Runtime.InteropServices;
+using Xunit;
+using FluentAssertions;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Cli.Utils.ScriptExecutorTests
+{
+    public class ScriptExecutorTests : TestBase
+    {
+        private static readonly string s_testProjectRoot = Path.Combine(AppContext.BaseDirectory, "TestAssets/TestProjects");
+
+        private TempDirectory _root;
+
+        public ScriptExecutorTests()
+        {
+            _root = Temp.CreateDirectory();
+        }
+
+        [Fact]
+        public void Test_Project_Local_Script_is_Resolved()
+        {
+            var sourceTestProjectPath = Path.Combine(s_testProjectRoot, "TestApp");
```

Comment:
Move these the next three lines to attributes and initialize them in the constructor. They are repeated through out the tests. And also, check for other instances of this happening in here.